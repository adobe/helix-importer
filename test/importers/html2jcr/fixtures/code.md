+-------------------------------------------------------------------------+
| Code                                                                    |
+=========================================================================+
| ```                                                                     |
| {                                                                       |
|   "type": "pwa-node",                                                   |
|   "name": "Action:csvimportmagento-0.0.1/push-product",                 |
|   "request": "launch",                                                  |
|   "runtimeExecutable": "${workspaceFolder}/node_modules/.bin/wskdebug", |
|   "envFile": "${workspaceFolder}/dist/.env.local",                      |
|   "timeout": 30000,                                                     |
|   "killBehavior": "polite",                                             |
|   "localRoot": "${workspaceFolder}",                                    |
|   "remoteRoot": "/code",                                                |
|   "outputCapture": "std",                                               |
|   "attachSimplePort": 0,                                                |
|   "runtimeArgs": [                                                      |
|     "csvimportmagento-0.0.1/push-product",                              |
|     "${workspaceFolder}/actions/push-product/index.js",                 |
|     "-v",                                                               |
|     "—disable-concurrency",                                             |
|     "—kind",                                                            |
|     "nodejs:12"                                                         |
|   ]                                                                     |
| },                                                                      |
| ...                                                                     |
|  "compounds": [                                                         |
|      {                                                                  |
|        "name": "Actions",                                               |
|        "configurations": [                                              |
|          ......                                                         |
|          "Action:csvimportmagento-0.0.1/push-product"                   |
|        ]                                                                |
|      },                                                                 |
|      {                                                                  |
|        "name": "WebAndActions",                                         |
|        "configurations": [                                              |
|          ......                                                         |
|          "Action:csvimportmagento-0.0.1/push-product"                   |
|        ]                                                                |
|      }                                                                  |
|    ]                                                                    |
| ```                                                                     |
+-------------------------------------------------------------------------+

+-------------------------------------------------------------------------+
| Code                                                                    |
+=========================================================================+
| Just plain text becomes richtext as well                                |
+-------------------------------------------------------------------------+
