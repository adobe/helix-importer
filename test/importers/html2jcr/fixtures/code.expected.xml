<?xml version="1.0" encoding="UTF-8"?>
<jcr:root xmlns:jcr="http://www.jcp.org/jcr/1.0" xmlns:nt="http://www.jcp.org/jcr/nt/1.0" xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:sling="http://sling.apache.org/jcr/sling/1.0" jcr:primaryType="cq:Page">
    <jcr:content cq:template="/libs/core/franklin/templates/page" jcr:primaryType="cq:PageContent" sling:resourceType="core/franklin/components/page/v1/page">
        <root jcr:primaryType="nt:unstructured" sling:resourceType="core/franklin/components/root/v1/root">
            <section sling:resourceType="core/franklin/components/section/v1/section" jcr:primaryType="nt:unstructured">
                <block sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Code" text="&lt;pre>&lt;code>{&#xa;  &quot;type&quot;: &quot;pwa-node&quot;,&#xa;  &quot;name&quot;: &quot;Action:csvimportmagento-0.0.1/push-product&quot;,&#xa;  &quot;request&quot;: &quot;launch&quot;,&#xa;  &quot;runtimeExecutable&quot;: &quot;${workspaceFolder}/node_modules/.bin/wskdebug&quot;,&#xa;  &quot;envFile&quot;: &quot;${workspaceFolder}/dist/.env.local&quot;,&#xa;  &quot;timeout&quot;: 30000,&#xa;  &quot;killBehavior&quot;: &quot;polite&quot;,&#xa;  &quot;localRoot&quot;: &quot;${workspaceFolder}&quot;,&#xa;  &quot;remoteRoot&quot;: &quot;/code&quot;,&#xa;  &quot;outputCapture&quot;: &quot;std&quot;,&#xa;  &quot;attachSimplePort&quot;: 0,&#xa;  &quot;runtimeArgs&quot;: [&#xa;    &quot;csvimportmagento-0.0.1/push-product&quot;,&#xa;    &quot;${workspaceFolder}/actions/push-product/index.js&quot;,&#xa;    &quot;-v&quot;,&#xa;    &quot;—disable-concurrency&quot;,&#xa;    &quot;—kind&quot;,&#xa;    &quot;nodejs:12&quot;&#xa;  ]&#xa;},&#xa;...&#xa; &quot;compounds&quot;: [&#xa;     {&#xa;       &quot;name&quot;: &quot;Actions&quot;,&#xa;       &quot;configurations&quot;: [&#xa;         ......&#xa;         &quot;Action:csvimportmagento-0.0.1/push-product&quot;&#xa;       ]&#xa;     },&#xa;     {&#xa;       &quot;name&quot;: &quot;WebAndActions&quot;,&#xa;       &quot;configurations&quot;: [&#xa;         ......&#xa;         &quot;Action:csvimportmagento-0.0.1/push-product&quot;&#xa;       ]&#xa;     }&#xa;   ]&#xa;&lt;/code>&lt;/pre>" model="code"/>
                <block_0 sling:resourceType="core/franklin/components/block/v1/block" jcr:primaryType="nt:unstructured" name="Code" text="&lt;p>Just plain text becomes richtext as well&lt;/p>" model="code"/>
            </section>
        </root>
    </jcr:content>
</jcr:root>